### DISCUSSION TITLE
conflict with svgr

### DISCUSSION THREAD

**Discussion by minsukim-29cm** [vite-pwa/vite-plugin-pwa#796] [Q&A]
![image](https://github.com/user-attachments/assets/499a1ebe-fdd7-4947-99aa-942f528040b6)

"""
vite.config.ts(9,5): error TS2769: No overload matches this call.
3:06:58 PM:   The last overload gave the following error.
3:06:58 PM:     Type 'Plugin' is not assignable to type 'PluginOption'.
3:06:58 PM:       Type 'Plugin' is not assignable to type 'Plugin<any>'.
3:06:58 PM:         Types of property 'resolveId' are incompatible.
"""

How can I resolve this confilct  about svgr and vitePWA?

**Comment by userquin** [comment]
Looks like the svgr plugin is for rollup and seems to be not compatible with your vite version.
